plugins/woocommerce-blocks/tests/e2e/tests/price-filter/price-filter.block_theme.side_effects.spec.ts

```diff
@@ -16,8 +16,8 @@ const blockData: BlockData< {
 		frontend: {},
 		editor: {},
 	},
-	urlSearchParamWhenFilterIsApplied: '?max_price=10',
-	endpointAPI: 'max_price=1000',
+	urlSearchParamWhenFilterIsApplied: '?max_price=5',
```

Comment:
Why do we need these 2 lines to change? I can't quite link this to the test above

Reply:
The changed part comes from the price filter. Sometimes, before running this test, a tax-related test was running. Depending on the tax settings, there are either 1 or 2 products when setting the max price to $10. By setting it to $5, there's only one product. I've discussed this with @gigitux in p1711023323470099/1711019541.954049-slack-C02UBB1EPEF.